dist/src/main/java/io/camunda/application/commons/service/CamundaServicesConfiguration.java

```diff
@@ -316,4 +323,20 @@ public AuthorizationChecker authorizationChecker(
       final AuthorizationSearchClient authorizationSearchClient) {
     return new AuthorizationChecker(authorizationSearchClient);
   }
+
+  @Bean
+  public ProcessCache processCache(
+      final GatewayRestConfiguration configuration,
+      final ProcessDefinitionServices processDefinitionServices,
```

Comment:
let's make sure this does not create any cyclic dependencies

Reply:
Should only happen if the process definition services also require the process cache (which isn't the case currently). Not sure if you'll get a warning or error at compile time in Spring for that setup already.